app/modules/sightings/schemas.py

```diff
@@ -98,25 +92,16 @@ class AugmentedEdmSightingSchema(BaseSightingSchema):
         DetailedAssetSchema,
         attribute='get_assets',
         many=True,
-        only=(
```

Comment:
As far as I can tell that list is the entirety of the DetailedAssetSchema so this is redundant

Reply:
I think we were omitting some things before. We must not need to anymore and the list grew to be the entirety.